src/golang/cmd/server/main.go

```diff
@@ -30,7 +34,41 @@ func main() {
 		*confPath = filepath.Join(cwd, "config", "server.yml")
 	}
 
-	log.SetFormatter(&log.TextFormatter{DisableQuote: true})
+	log.SetFormatter(&log.TextFormatter{
+		DisableQuote: true,
+		ForceColors:  true,
+	})
+
+	// Always store all logs to a log file.
+	// With lumberjack.Logger we can do log rotation to prevent it from growing infinitely.
+	log.SetOutput(&lumberjack.Logger{
+		Filename:   serverLogPath,
+		MaxSize:    100, // megabytes
+		MaxBackups: 3,
+		MaxAge:     28, // days
```

Comment:
Any reason why 28 and not 30?

Reply:
The example code had 28. Maybe because Feb only have 28 days. :) I think it's fine though.